src/components/combo_box/_combo_box.scss

```
@@ -24,6 +24,14 @@
     // to override the padding added above
     @include euiFormControlLayoutPadding(1); /* 2 */
 
+    .euiComboBoxPill {
+      $inputMinWidth: 14px;
```

Could we try to use an Eui variable here? We could do `$euiSize` and then reduce the extra margin below from 4 to 2px and still be subtracting 18px.
```suggestion
      $inputMinWidth: $euiSize;
```